README.md

```diff
@@ -1,39 +1,124 @@
+* Public [PhET.io Developer Guide](https://phet-io.colorado.edu/devguide/#documentation)
+* The [Open Sound Control 1.0 Specification](http://opensoundcontrol.org/spec-1_0)
+* [osc.js](https://github.com/colinbdclark/osc.js)
+
+## Architecture
+
+The PhET OSC Bridge includes several components:
+
+1. A client-side JavaScript component, the <code>phetosc.bridge</code> that can be added to any PhET-iO wrapper HTML page to listen for sim events.
+2. A web server that serves up PhET-iO wrapper HTML pages, and responds to incoming Web Socket requests from the client-side bridge component.
+3. An OSC output port (configured to use UDP by default) that relays OSC messages to the the remote server (e.g. SuperCollider or Max).
+
+## Using the PhET OSC Bridge
+
+### Getting Set Up
+First, follow the instructions provided with your PhET-iO license to generate an appropriate wrapper HTML page. More information is available in the [PhET-iO Developer's Guide](https://phet-io.colorado.edu/devguide/).
+
+Secondly, install the <code>phet-osc-bridge</code> npm module:
+
+    npm install -g phet-osc-bridge
+
+### Adding the Bridge Component to your PhET-iO Wrapper
+The PhET OSC Bridge web server will serve up the required client JavaScript file. You just need to add it to your PhET-iO wrapper's head using a <code>script</code> tag:
+
+    <script type="text/javascript" src="/phet-osc-bridge/phet-osc-bridge.js"></script>
+
+Next, you'll need to instantiate the bridge component and bind it to the PhET sim's <code>simIFrameClient</code> object. This will cause all events emitted from the sim to be available across the bridge. Paste this code into the <code>onSimInitialized</code> event handler function that is included in your wrapper's call to <code>simIFrameClient.launchSim()</code>:
+
+    var oscBridge = phetosc.bridge();
+    oscBridge.bind(simIFrameClient);
+
+### Starting the Bridge's Server
+Lastly, you'll need to start the PhET OSC Bridge server. In your terminal, navigate to the directory in which your PhET-iO wrapper files are located, and run:
+
+    phet-osc-bridge --phetPath .
+
+By default, your wrapper file will be available in your web browser at ``http://localhost:8081/<wrapper-file-name>``.
+
+## Server Configuration Options
+Other configuration options can be specified on the command line. More information can be found by running the <code>phet-osc-bridge</code> command with the <code>--help</code> flag.
+
+## Bridge Options
+
+The <code>phetosc.bridge</code> component provides a variety of configuration options, which are described in the sections below.
+
+### Filtering Events
+In many cases, only a subset of events are relevant for a sonification. The <code>phetioIDPatterns</code> option allows you to specify an array of regular expression patterns that will be matched against the <code>phetioID</code> of events. Only those events with a <code>phetioID</code> matching the specified list of regular expressions will be sent via OSC.
```

Comment:
Comment could be clarified to explain that the patterns will be ORed together as per https://github.com/fluid-studios/phet-osc-bridge/pull/6/files#diff-8621dab8da352b7954438b801f34de0cR51 - that is, it is sufficient to match any one of the expressions